src/frontend/Komponenter/Behandling/Vurdering/VisEllerEndreVurdering.tsx

```diff
@@ -114,7 +116,8 @@ const VisEllerEndreVurdering: FC<Props> = ({
                     lagreVurdering={lagreVurdering}
                     feilmelding={feilmelding || resetFeilmelding}
                     settRedigeringsmodus={settRedigeringsmodus}
-                    resetVurdering={resetVurdering}
+                    settVurderingVarTom={settVurderingVarTom}
+                    vurderingVarTom={vurderingVarTom}
```

Comment:
`const initiellRedigeringsmodus = vurdering.resultat === Vilkårsresultat.IKKE_TATT_STILLING_TIL) ?
Redigeringsmodus.IKKE_PÅSTARTET:  Redigeringsmodus.VISNING;`

Kan vi sende med `initiellRedigeringsmodus={initiellredigeringsmodus}` istedenfor å ha en egen state, eller vil det bli for "puslete"?

Reply:
Joda, det funker det. Smart tenkt! Fikset i 00daad7